job_bundles/tile_render_maya_ffmpeg/template.yaml

```diff
@@ -0,0 +1,331 @@
+specificationVersion: jobtemplate-2023-09
+name: tile render maya ffmpeg
+parameterDefinitions:
+- name: MayaSceneFile
```

Comment:
To make it easier to read and understand the parameters, some of our samples use a blank line to separate the differently grouped parameters, and start each block with a comment for the group label. See the parameters in https://github.com/aws-deadline/deadline-cloud-samples/blob/mainline/job_bundles/blender_render/template.yaml for an example.